apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -368,6 +368,10 @@ const AvailabilityPage = ({ profile, eventType }: Props) => {
     ),
     [timeZone]
   );
+  const currentPath = router.asPath.split("/");
+  currentPath.pop();
+  currentPath.shift();
+  const slug = currentPath.join("/");
```

Comment:
Will this work with queries?

Reply:
Do we use this page with queries alone? Can you give me an example?
I would probably need to skip the execution of this and the HeadSEO code if we run this page on queries alone I guess

Reply:
The date is basically set from a query. Hence my question

Reply:
[See my comment](https://github.com/calcom/cal.com/pull/3255#pullrequestreview-1034536791)